config/22.7/make.conf

```
@@ -78,7 +78,7 @@ net_asterisk16_UNSET=		DAHDI XMPP
 net_freeradius3_SET=		LDAP MITKRB_PORT MYSQL SQLITE3
 net_freeradius3_UNSET=		HEIMDAL
 net_frr7_SET=			MULTIPATH SNMP
-net_haproxy24_SET=		LUA
+net_haproxy24_SET=		LUA PROMEX
```

```suggestion
net_haproxy24_SET=		LUA # PROMEX
```

maybe if we settle on this you can review this as necessary and the script is silent?

---

LGTM :)